package.json

```diff
@@ -19,8 +19,8 @@
     "node": ">=18.15.0"
   },
   "dependencies": {
-    "chalk": "^4.1.2",
-    "lit": "^2.6.1"
+    "chalk": "^5.2.0",
```

Comment:
Updating this to v5+, there are additional things that need to be updated as well. Please see the following files from the generator. 

1. https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/generateDocs.js
2. https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/pre-commit.mjs